numpy/core/numerictypes.py

```diff
@@ -485,9 +484,9 @@ def sctype2char(sctype):
 
     Examples
     --------
-    >>> for sctype in [np.int32, float, complex, np.string_, np.ndarray]:
+    >>> for sctype in [np.int32, np.double, complex, np.string_, np.ndarray]:
     ...     print(np.sctype2char(sctype))
-    l
+    i
```

Comment:
This is platform-dependent, since int32 is the first of `long` and `int` that's the right size. Either use `np.int_` above, and `l` here, or just mark this as may vary.